onnxruntime/core/providers/dml/OperatorAuthorHelper/MLOperatorAuthorPrivate.h

```diff
@@ -58,10 +58,10 @@ IMLOperatorKernelCreationContextPrivate : public IMLOperatorKernelCreationContex
 };
 
 interface __declspec(uuid("1d2e1226-a918-4236-8775-175cf1f52c9a"))
-IMLOperatorKernelCreationContextPrivate1 : public IMLOperatorKernelCreationContextPrivate 
+IMLOperatorKernelCreationContextNodeWrapperPrivate : public IMLOperatorKernelCreationContextPrivate
 {
     //! Gets the minimum size of a char buffer to store the node name (including null terminator).
-    //! Returns 0 if the node has no name.
+    //! Returns 1 if the node has no name (calling GetUtf8Name will write a single null terminator).
     STDMETHOD_(uint32_t, GetUtf8NameSizeInBytes)() const noexcept PURE;
  
```

Comment:
How about \_\_BufferSize\_\_? Typically any Win32 function of the naming convention "GetThingSize/Length" (e.g. `IDWriteLocalizedStrings::GetStringLength` or `hstring::size`) only returns the length of the actual thing, excluding any trailing terminator, whereas GetThing*Buffer*Size does include the terminator.